source/Introduction.rst

```diff
@@ -1,11 +1,16 @@
-.. _InstallationGuide:
-
 Introduction
 ============
 
-TODO:
+Space ROS is an open-source spacecraft flight software framework.
+It is designed to be platform independent, portable and project independent.
+Derived from the ROS 2 framework and conforming to the ROS 2 Application Programming Interface (API), Space ROS will be hardened to be compatible with the demands of safety-critical space robotics applications.
+Space ROS will provide a robust framework for space robotic applications where ROS 2 applications can be reused with little to no modification, enabling the space community to take advantage of the innovation of the ROS community.
+This will shorten the time for development of novel space robotics capabilities, enable reuse of capabilities between missions, and lower the life-cycle cost of new robotic missions.
```

Comment:
Again, I'm unsure if robotics should always be pluralized but I'll suggest it as a discretionary change.

```suggestion
This will shorten the time for development of novel space robotics capabilities, enable reuse of capabilities between missions, and lower the life-cycle cost of new robotics missions.
```

Reply:
Yes, we clearly need to be careful. There is "space robotics" as a general field and "robotic" as a adjective (robotic arm). I've updated to use "robotics missions" here.